config/environments/production.rb

```diff
@@ -68,13 +76,13 @@
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.perform_deliveries = true
   config.action_mailer.smtp_settings = {
-    address: 'smtp.mandrillapp.com',
-    port: 587,
+    address: ENV['MAIL_SMTP_HOST'] || MAIL_DEFAULTS[:host],
```

Comment:
let's name MAIL_DEFAULST key exactly the same as it is accepted here -- `:address`